src/Microsoft.IdentityModel.Validators/AadIssuerValidator/AadIssuerValidator.cs

```diff
@@ -180,18 +191,24 @@ internal string AadAuthorityV2
 
             try
             {
-                BaseConfigurationManager effectiveConfigurationManager = GetEffectiveConfigurationManager(securityToken);
-                if (validationParameters.RefreshBeforeValidation)
-                    effectiveConfigurationManager.RequestRefresh();
+                var isV2Issuer = IsV2Issuer(securityToken);
 
-                BaseConfiguration configuration = await effectiveConfigurationManager.GetBaseConfigurationAsync(CancellationToken.None).ConfigureAwait(false);
-                string aadIssuer = configuration.Issuer;
+                var effectiveConfigurationManager = GetEffectiveConfigurationManager(isV2Issuer);
+                var baseConfiguration = await GetBaseConfiguration(effectiveConfigurationManager, validationParameters).ConfigureAwait(false);
+                string aadIssuer = baseConfiguration.Issuer;
```

Comment:
can we just pass baseConfiguration.Issuer rather than create this local variable?

Reply:
I've readjusted this part of the method for readability and ended up using this variable more